econml/_ortho_learner.py

```diff
@@ -770,18 +771,19 @@ def _fit_nuisances(self, Y, T, X=None, W=None, Z=None, sample_weight=None, group
         return nuisances, fitted_models, fitted_inds, scores
 
     def _fit_final(self, Y, T, X=None, W=None, Z=None, nuisances=None, sample_weight=None,
-                   freq_weight=None, sample_var=None):
+                   freq_weight=None, sample_var=None, groups=None):
         self._ortho_learner_model_final.fit(Y, T, **filter_none_kwargs(X=X, W=W, Z=Z,
                                                                        nuisances=nuisances,
                                                                        sample_weight=sample_weight,
                                                                        freq_weight=freq_weight,
-                                                                       sample_var=sample_var))
+                                                                       sample_var=sample_var,
+                                                                       groups=groups))
         self.score_ = None
         if hasattr(self._ortho_learner_model_final, 'score'):
             self.score_ = self._ortho_learner_model_final.score(Y, T, **filter_none_kwargs(X=X, W=W, Z=Z,
                                                                                            nuisances=nuisances,
-                                                                                           sample_weight=sample_weight)
-                                                                )
+                                                                                           sample_weight=sample_weight,
+                                                                                           groups=groups))
 
     def const_marginal_effect(self, X=None):
```

Comment:
when I call cons_marginal_effecT_inference I get this bad printing:
![image](https://user-images.githubusercontent.com/13246065/128015668-cecd32e7-a24f-4bf2-9b0c-79901612b5f6.png)

Reply:
It's a weird effect from Pandas. You can get the same effect with:
```Python
pd.DataFrame({"a":["$(T0)_0$", "$(T1)_0$"]})
```
However, this:
```Python
pd.DataFrame({"a":["(T0)$_0$", "(T1)$_0$"]})
```
looks ok. I will change it to this.